ui/src/experiments/components/ExperimentCollaborationArticlesLink.jsx

```diff
@@ -4,16 +4,16 @@ import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
 import { LITERATURE } from '../../common/routes';
 
-function ExperimentCollaborationArticlesLink({ recordId }) {
+function ExperimentCollaborationArticlesLink({ collaboration }) {
   return (
-    <Link to={`${LITERATURE}?q=collaboration.value:${recordId}`}>
+    <Link to={`${LITERATURE}?q=collaboration:${collaboration.get('value')}`}>
```

Comment:
shouldn't be ``collaborations.value``?  Is ``collaboration`` part of query parser? I cannot find ``collaboration``  in our [mappings](https://github.com/inspirehep/inspirehep/blob/master/backend/inspirehep/search/mappings/v7/records/hep.json#L592)

Reply:
It's part of the query parser